pkg/util/namespace_test.go

```diff
@@ -0,0 +1,264 @@
+package util
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+	coretesting "k8s.io/client-go/testing"
+)
+
+func TestIsNamespaceExist(t *testing.T) {
+	type args struct {
+		client    kubernetes.Interface
```

Comment:
```suggestion
		client    *fake.Clientset
```
If use `*fake.Clientset` here, then you don't need to convert with `tt.args.client.(*fake.Clientset)`.

Reply:
Ok, it's modified, thanks for pointing it out